languages/python/oso/tests/test_polar_roles.py

```diff
@@ -2538,105 +2201,6 @@ def test_actor_can_assume_role(init_oso, sample_data):
     #     assert repo.org_id == "osohq"
 
 
-@pytest.mark.skip("TODO: validation / this isn't our problem anymore")
-def test_mismatched_id_types_throws_error(engine, Base, User):
-    pass
-    # class One(Base):
-    #     __tablename__ = "ones"
-    #
-    #     id = Column(String(), primary_key=True)
-    #
-    # class Two(Base):
-    #     __tablename__ = "twos"
-    #
-    #     id = Column(Integer(), primary_key=True)
-    #
-    # Session = sessionmaker(bind=engine)
-    #
-    # oso = SQLAlchemyOso(Base)
-    #
-    # with pytest.raises(OsoError):
-    #     oso.enable_roles(User, Session)
-
-
-@pytest.mark.skip("TODO: not a thing anymore, maybe?")
-def test_enable_roles_twice(engine, Base, User):
```

Comment:
should be fine since everything's idempotent, but maybe test (that it's fine) as a follow-up?